homeassistant/components/teslemetry/icons.json

```diff
@@ -0,0 +1,19 @@
+{
+  "entity": {
+    "climate": {
+      "driver_temp": {
+        "name": "[%key:component::climate::title%]",
```

Comment:
```suggestion
```

Reply:
Thanks, I just copied from `strings.json`. I'll remove that.